### PR TITLE
Add YAML for testproject-plugin

### PR DESCRIPTION
**Pull request by eldarkra**
# Description

Git repository:
https://github.com/jenkinsci/testproject-plugin
Hosting issue
https://issues.jenkins-ci.org/browse/HOSTING-689

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by eldarkra** [comment]
Everything was fixed. Please review, 10x

File: permissions/plugin-testproject.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "testproject"
+github: "jenkinsci/testproject-plugin"
+paths:
+- "org/jenkins-ci/plugins/testproject"
```
Comment (batmat) [inline permissions/plugin-testproject.yml]: https://github.com/jenkinsci/testproject-plugin/blob/master/pom.xml#L10-L11

Comment (eldarkra) [reply permissions/plugin-testproject.yml]: @batmat, fixed.

Comment (batmat) [reply permissions/plugin-testproject.yml]: Note that in this case, the more correct fix would have simply been to remove the `groupId` override. But it gives the same. So LGTM

**Pull request merged**